src/test/java/dev/jbang/cli/TestRun.java

```diff
@@ -560,6 +560,25 @@ void testFetchFromGist(@TempDir Path dir) throws IOException {
 		assertEquals(x.getFileName().toString(), "checklabeler.java");
 	}
 
+	@Test
+	void testFetchFromGistWithFilename(@TempDir Path dir) throws IOException {
+
+		String u = Util.swizzleURL("https://gist.github.com/tivrfoa/503c88fb5123b1000c37a3f2832d4773#file-file2-java");
```

Comment:
test in your account is fine - if/when we need to change it we would need to fork it anyway; so doesn't matter as long as you don't delete it :)